### ISSUE TITLE
Improve the documentation and emails for password resets

### ISSUE THREAD

**Issue by ccostino** [cloud-gov/product#1386]
In order to make it clearer and easier to understand the password reset process, we need to update our account management documentation and password reset verification emails to clearly state the steps a customer must take.

We recently had a support request come in stating that the password reset workflow wasn't working.  It is, but the verification email simply states this:

>Hello!
>
>You've requested to reset your password for cloud.gov. (If you didn't request this, you can ignore this email.)
>
>Your next steps
>
>Verify your email using this link `<generated link>`, and follow those instructions to generate your temporary password. Save your generated password before following the next step to log in with that password.
>After you log in, you can change your password using this link https://account.fr.cloud.gov/change-password.
>If you run into problems or have any questions, please email us at cloud-gov-support@gsa.gov.
>
>Thank you,
>The cloud.gov team

The steps more accurately laid out are this:
1. Click on the reset your password link
1. Enter your email address to receive the reset password confirmation email
1. Click on the verification link in the first step of that email
1. Enter your email address again on the verification page that loads
1. Receive the temporary password once your email is verified
1. Login to your cloud.gov account at https://login.fr.cloud.gov/ with the temporary password
1. After logging in, go to https://account.fr.cloud.gov/change-password to change the password (you would enter the temporary password as the old password, and then your new password to change to)

Furthermore, as a user goes through the screens and clicks the links, the flow itself is still not quite clear.  Upon verifying your email address you're presented with a temporary password and links to both login and change your password, but you should only login first with the temporary password.  Changing the password shouldn't happen until after you've logged in.

## Acceptance Criteria
* [x] GIVEN a need to reset a password
  WHEN a user views our account documentation
  AND follows the instructions
  THEN their password reset request should be successful
* [x] GIVEN a password reset request
  WHEN a user receives a verification email
  AND follows the instructions
  THEN their password reset should be successful
--- 

## Security considerations
None; this is a public-facing standard user workflow

## Implementation sketch
[links to background notes, sketches, and/or relevant documentation
* [X] [Our accounts documentation](https://cloud.gov/docs/getting-started/accounts/) should clarify this process and clearly lay out the corrects steps
  * See https://github.com/cloud-gov/cg-site/pull/1726
* [X] The language and steps given in the verification email should be much clearer
  * See PR https://github.com/cloud-gov/cg-uaa-extras/pull/168
* [x] If possible (or likely as a separate ticket), the screens in UAA for the various pages involved in this workflow should be improved so the language is clearer as well
  * See stubbed out issue https://github.com/cloud-gov/cg-uaa-extras/issues/169

**Comment by mheadd** [comment]
Do we know if we can include graphics in the emails to the customers? That might help clarify things.

**Comment by apburnes** [comment]
I don't see why not.  @mheadd are you thinking of screenshots of the workflow or something more like an infographic?

**Comment by bengerman13** [comment]
I'd like to see us make this workflow more intuitive, rather than document it better. Documentation is probably easier, so maybe both.

**Comment by apburnes** [comment]
Making the workflow more intuitive is the end goal. I'll update documentation to bridge gap until we can get it updated and create an issue outlining next steps.

**Comment by bengerman13** [comment]
I *think* a big win on the documentation would be to make a numbered list of steps, so folks know that there are more steps after the first link.

**Comment by apburnes** [comment]
Waiting on PR https://github.com/cloud-gov/cg-uaa-extras/pull/168 to close
